spec-main/fixtures/snapshots/native-window-open.snapshot.txt

```diff
@@ -2,23 +2,18 @@
   [
     "top=5,left=10,resizable=no",
     {
-      "sender": "[WebContents]",
-      "returnValue": 2
+      "sender": "[WebContents]"
     },
     "about:blank",
     "frame name",
     "new-window",
     {
       "show": true,
-      "width": 800,
+      "title": "cool",
+      "width": 200,
       "height": 600,
-      "webContents": "[WebContents]",
-      "top": "5",
```

Comment:
Note the wrong type here, not that it matters for the BrowserWindow constructor (as it will use x, y)